packages/core/src/App/Components/Layout/Header/toggle-menu-drawer.jsx

```diff
@@ -21,6 +22,8 @@ const MenuLink = ({
     text,
     onClickLink,
 }) => {
+    const derivStaticUrl = /(deriv.com)/.test(link_to);
```

Comment:
1) This won't work on test links. Why are we checking this here? If we need this, we should make sure that it works on test links too.
2) it should be `deriv_static_url` as per our convention?

Reply:
It does works for test links. Just tested. Will change the naming convention.